modules/gapi/test/streaming/gapi_streaming_tests.cpp

```diff
@@ -459,6 +459,24 @@ TEST_P(GAPI_Streaming, SmokeTest_VideoConstSource_NoHang)
     EXPECT_EQ(ref_frames, test_frames);
 }
 
+TEST_P(GAPI_Streaming, SmokeTest_Queue_Capacity_1)
+{
+    auto cc = cv::GComputation([](){
+        cv::GMat in;
+        return cv::GComputation(in, cv::gapi::copy(in));
+    }).compileStreaming(cv::compile_args(cv::gapi::streaming::queue_capacity{1}));
+
+    auto path = findDataFile("cv/video/768x576.avi");
+    try {
+        cc.setSource(gapi::wip::make_src<cv::gapi::wip::GCaptureSource>(path));
+    } catch(...) {
+        throw SkipTestException("Video file can not be opened");
+    }
+    cv::Mat out_mat;
+    cc.start();
+    while (cc.pull(cv::gout(out_mat))) { };
+}
```

Comment:
I don't get it, this is not what I've been asking for

Reply:
Done